core/audits/dobetterweb/doctype.js

```diff
@@ -94,7 +116,7 @@ class Doctype extends Audit {
       };
     } else {
       return {
-        score: 1,
+        score: 0,
```

Comment:
I think `UIStrings.explanationWrongDoctype` works as a catch-all fallback here, we probably don't need this extra if-else case.